product/fhir-data.mdx

```diff
@@ -47,6 +47,8 @@ Base element is a [Bundle](https://www.hl7.org/fhir/R4/bundle.html) resource, in
 | [DocumentReference](https://www.hl7.org/fhir/R4/documentreference.html) (Notes)       | [US Core DocumentReference](https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-documentreference.html)                                   | Includes reference to encounter. Configurable to show all historical notes or just those associated with an encounter on the PDF. If the latter, the notes truncation feature is enabled by default. |
 | [Condition](https://www.hl7.org/fhir/R4/condition.html) (Problems)                    | [US Core Condition Problems Health Concerns](https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-condition-problems-health-concerns.html) | Includes all historical problems, not just what is on Connective Health’s patient summary template.                                                                                                  |
 | [Procedure](https://www.hl7.org/fhir/R4/procedure.html)                               | [US Core Procedure](https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-procedure.html)                                                   | Based on Connective Health's patient summary template. Includes reference to encounter.                                                                                                              |
+| [DiagnosticReport](https://www.hl7.org/fhir/R4/diagnosticreport.html)                 | [US Core DiagnosticReport](https://www.hl7.org/fhir/us/core/StructureDefinition/us-core-diagnosticreport-note)                                     | Based on Connective Health's patient summary template. Includes embedded Observation Clinical Results.                                                                                               |
```

Comment:
I think we probably should provide a bit more of an example or clarity as to what these resources will contain, i.e. imaging reports.